lambda/README.md

```diff
@@ -53,6 +53,10 @@ Creates an AWS Lambda function
 
     Path to the event handler
 
+* `image` (`string`, default: `null`)
+
+    URI of a container image with the Lambda's source. Either `package_path`, `package_s3` or `image` is required.
+
 * `layer_qualified_arns` (`list(string)`, default: `[]`)
```

Comment:
Can Lambda layers coexist with the container based function? If not, should we care about some validation for this?

Reply:
I'd expect AWS to throw an error if you try to use an image along with lambda layers, but haven't tested that yet.

Reply:
fixed in 46c2f69